src/cortex_cli/cortex_cli.py

```diff
@@ -378,6 +362,55 @@ def _validate_cortex_cli_auth_login(no_daemon, no_refresh, config_file) -> dict:
     return config
 
 
+def _refresh_tokens(  # pylint: disable=too-many-arguments
+    refresh_period: int,
+    no_daemon: bool,
+    no_refresh: bool,
+    config: ConfigFile,
```

Comment:
`auth_server_url`, `realm` and `client_id` are stored in `config`, why are they specified here also separately?

Reply:
Good point, I relied on "extract method" 😅 , will fix it.

Reply:
It probably doesn't need the `disable=too-many-arguments` directive any more.